lab/requirements.md

```diff
@@ -0,0 +1,53 @@
+# OPI Lab Requirements
+
+## Access
+
+- Lab staff can access to all machines.
+- OPI members should access only to their own machines.
+- Remote access to a machine requires private VPN specific to the machine.
+- OPI members cannot access to a machine from another one.
+- SSH connection is available through private VPN.
+- Emergency console can be enabled for remote access in case of issue.
+
+## Backup
+
+- All servers must be backed up.
```

Comment:
Absolutely not. We need machines to be cattle and not pets, meaning we can wipe them and reinstall with configuration stored in a git repository via something like ansible

Reply:
Original requirement had the "critical" adjective:

<html>
<body>
<!--StartFragment-->

All critical servers backed up | BACKUP | COMMON |   |  
-- | -- | -- | -- | --
Restore of critical servers must be done within a couple of hours | BACKUP | COMMON |   |  
Team in place trained to do backup/restore | BACKUP | COMMON

<!--EndFragment-->
</body>
</html>

At least I do not yet have visibility on what constitutes critical servers.   Could be a database of lab inventory, could be  a wiki, mirror of Linux distributions, etc.  Anything that has long-lived state.  Who knows.  If the set of critical servers is currently the empty set set, so be it.  Keep that requirement as is so if OPI decides to keep important state in the lab, it is covered.  I agree most of the lab should house stateless ephemeral servers that can be wiped out at any time.

Reply:
I understand your point of view. You want to back up configuration in git for normal machines. And you would keep a full backup option for some special machines if any.